package.json

```diff
@@ -61,10 +65,12 @@
   "devDependencies": {
     "@commitlint/cli": "^7.1.2",
     "@commitlint/config-conventional": "^7.1.2",
+    "@types/webpack": "^4.4.24",
```

Comment:
this shouldn't be on deps as well? Won't this create the same errors on consumers?

Reply:
These types are not used in the index.d.ts so they do not need to be deps

Reply:
webpack was referenced, so I moved that up to deps